<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="22" passed="17" failed="4" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-11-26T22:37:05 MMT" name="Default suite" finished-at="2020-11-26T22:43:51 MMT" duration-ms="406429">
    <groups>
      <group name="QA">
        <method signature="DependsOnGroupTest.test2_qa_regression()[pri:0, instance:group.DependsOnGroupTest@1677d1]" name="test2_qa_regression" class="group.DependsOnGroupTest"/>
      </group> <!-- QA -->
      <group name="regression">
        <method signature="DependsOnGroupTest.test2_qa_regression()[pri:0, instance:group.DependsOnGroupTest@1677d1]" name="test2_qa_regression" class="group.DependsOnGroupTest"/>
      </group> <!-- regression -->
      <group name="smoke">
        <method signature="DependsOnGroupTest.test1_smoke()[pri:0, instance:group.DependsOnGroupTest@1677d1]" name="test1_smoke" class="group.DependsOnGroupTest"/>
      </group> <!-- smoke -->
    </groups>
    <test started-at="2020-11-26T22:37:05 MMT" name="Default test" finished-at="2020-11-26T22:43:51 MMT" duration-ms="406429">
      <class name="priority.PriorityTest">
        <test-method signature="signIn()[pri:1, instance:priority.PriorityTest@1ffe63b9]" started-at="2020-11-26T22:42:45 MMT" name="signIn" finished-at="2020-11-26T22:42:45 MMT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signIn -->
        <test-method signature="searchDress()[pri:2, instance:priority.PriorityTest@1ffe63b9]" started-at="2020-11-26T22:43:37 MMT" name="searchDress" finished-at="2020-11-26T22:43:37 MMT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- searchDress -->
        <test-method signature="addToCart()[pri:3, instance:priority.PriorityTest@1ffe63b9]" started-at="2020-11-26T22:43:41 MMT" name="addToCart" finished-at="2020-11-26T22:43:41 MMT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addToCart -->
        <test-method signature="makePayment()[pri:4, instance:priority.PriorityTest@1ffe63b9]" started-at="2020-11-26T22:43:51 MMT" name="makePayment" finished-at="2020-11-26T22:43:51 MMT" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- makePayment -->
        <test-method signature="paymentHistory()[pri:5, instance:priority.PriorityTest@1ffe63b9]" started-at="2020-11-26T22:43:51 MMT" name="paymentHistory" finished-at="2020-11-26T22:43:51 MMT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- paymentHistory -->
        <test-method signature="signOut()[pri:6, instance:priority.PriorityTest@1ffe63b9]" started-at="2020-11-26T22:43:51 MMT" name="signOut" finished-at="2020-11-26T22:43:51 MMT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signOut -->
      </class> <!-- priority.PriorityTest -->
      <class name="demo_project.DemoProjectTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:demo_project.DemoProjectTest@51e5fc98]" started-at="2020-11-26T22:42:45 MMT" name="setUp" finished-at="2020-11-26T22:43:29 MMT" duration-ms="44183" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="signIn()[pri:1, instance:demo_project.DemoProjectTest@51e5fc98]" started-at="2020-11-26T22:43:29 MMT" name="signIn" finished-at="2020-11-26T22:43:37 MMT" duration-ms="7818" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signIn -->
        <test-method signature="searchProduct()[pri:2, instance:demo_project.DemoProjectTest@51e5fc98]" started-at="2020-11-26T22:43:37 MMT" name="searchProduct" finished-at="2020-11-26T22:43:41 MMT" duration-ms="4656" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- searchProduct -->
        <test-method signature="signOut()[pri:3, instance:demo_project.DemoProjectTest@51e5fc98]" started-at="2020-11-26T22:43:41 MMT" name="signOut" finished-at="2020-11-26T22:43:44 MMT" duration-ms="3009" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signOut -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:demo_project.DemoProjectTest@51e5fc98]" started-at="2020-11-26T22:43:44 MMT" name="tearDown" finished-at="2020-11-26T22:43:51 MMT" duration-ms="6931" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- demo_project.DemoProjectTest -->
      <class name="dependency.DependencyTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:dependency.DependencyTest@7c469c48]" started-at="2020-11-26T22:40:10 MMT" name="setUp" finished-at="2020-11-26T22:41:24 MMT" duration-ms="74037" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="signIn()[pri:0, instance:dependency.DependencyTest@7c469c48]" started-at="2020-11-26T22:41:24 MMT" name="signIn" finished-at="2020-11-26T22:41:32 MMT" duration-ms="8204" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signIn -->
        <test-method signature="searchProduct()[pri:0, instance:dependency.DependencyTest@7c469c48]" started-at="2020-11-26T22:41:32 MMT" name="searchProduct" finished-at="2020-11-26T22:41:34 MMT" depends-on-methods="dependency.DependencyTest.signIn" duration-ms="1877" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- searchProduct -->
        <test-method signature="pageNameVerification()[pri:0, instance:dependency.DependencyTest@7c469c48]" started-at="2020-11-26T22:41:34 MMT" name="pageNameVerification" finished-at="2020-11-26T22:41:34 MMT" depends-on-methods="dependency.DependencyTest.searchProduct, dependency.DependencyTest.signIn" duration-ms="7" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Search - My Store] but found [Login - My Store]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Search - My Store] but found [Login - My Store]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at dependency.DependencyTest.pageNameVerification(DependencyTest.java:70)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- pageNameVerification -->
        <test-method signature="signOut()[pri:0, instance:dependency.DependencyTest@7c469c48]" started-at="2020-11-26T22:41:34 MMT" name="signOut" finished-at="2020-11-26T22:41:34 MMT" depends-on-methods="dependency.DependencyTest.signIn, dependency.DependencyTest.pageNameVerification" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method DependencyTest.signOut()[pri:0, instance:dependency.DependencyTest@7c469c48] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method DependencyTest.signOut()[pri:0, instance:dependency.DependencyTest@7c469c48] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:102)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signOut -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:dependency.DependencyTest@7c469c48]" started-at="2020-11-26T22:41:34 MMT" name="tearDown" finished-at="2020-11-26T22:41:47 MMT" duration-ms="12747" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- dependency.DependencyTest -->
      <class name="assertions.AssertionTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:assertions.AssertionTest@12e61fe6]" started-at="2020-11-26T22:41:47 MMT" name="setUp" finished-at="2020-11-26T22:42:38 MMT" duration-ms="50743" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testPageNameVerification()[pri:0, instance:assertions.AssertionTest@12e61fe6]" started-at="2020-11-26T22:42:38 MMT" name="testPageNameVerification" finished-at="2020-11-26T22:42:40 MMT" duration-ms="1840" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testPageNameVerification -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:assertions.AssertionTest@12e61fe6]" started-at="2020-11-26T22:42:40 MMT" name="tearDown" finished-at="2020-11-26T22:42:45 MMT" duration-ms="4926" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- assertions.AssertionTest -->
      <class name="dataProvider.DataProviderTest">
        <test-method signature="login(java.lang.String, java.lang.String)[pri:0, instance:dataProvider.DataProviderTest@7ee955a8]" started-at="2020-11-26T22:37:05 MMT" name="login" data-provider="dp_logInData" finished-at="2020-11-26T22:38:23 MMT" duration-ms="77961" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[testimonial@yopmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[test@123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The Actual & Expected Results Match expected [My account - My Store] but found [Login - My Store]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The Actual & Expected Results Match expected [My account - My Store] but found [Login - My Store]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at dataProvider.DataProviderTest.login(DataProviderTest.java:30)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="login(java.lang.String, java.lang.String)[pri:0, instance:dataProvider.DataProviderTest@7ee955a8]" started-at="2020-11-26T22:38:23 MMT" name="login" data-provider="dp_logInData" finished-at="2020-11-26T22:39:07 MMT" duration-ms="44034" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[invalid@yopmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[password]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The Actual & Expected Results Match expected [My account - My Store] but found [Login - My Store]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The Actual & Expected Results Match expected [My account - My Store] but found [Login - My Store]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at dataProvider.DataProviderTest.login(DataProviderTest.java:30)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="login(java.lang.String, java.lang.String)[pri:0, instance:dataProvider.DataProviderTest@7ee955a8]" started-at="2020-11-26T22:39:07 MMT" name="login" data-provider="dp_logInData" finished-at="2020-11-26T22:40:10 MMT" duration-ms="62310" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[testng@yopmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[test@123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The Actual & Expected Results Match expected [My account - My Store] but found [Login - My Store]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The Actual & Expected Results Match expected [My account - My Store] but found [Login - My Store]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at dataProvider.DataProviderTest.login(DataProviderTest.java:30)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
      </class> <!-- dataProvider.DataProviderTest -->
      <class name="group.DependsOnGroupTest">
        <test-method signature="test1_smoke()[pri:0, instance:group.DependsOnGroupTest@1677d1]" started-at="2020-11-26T22:41:47 MMT" name="test1_smoke" finished-at="2020-11-26T22:41:47 MMT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1_smoke -->
        <test-method signature="test2_qa_regression()[pri:0, instance:group.DependsOnGroupTest@1677d1]" started-at="2020-11-26T22:41:47 MMT" name="test2_qa_regression" finished-at="2020-11-26T22:41:47 MMT" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test2_qa_regression -->
        <test-method signature="test3()[pri:0, instance:group.DependsOnGroupTest@1677d1]" started-at="2020-11-26T22:41:47 MMT" name="test3" depends-on-groups="regression" finished-at="2020-11-26T22:41:47 MMT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test3 -->
      </class> <!-- group.DependsOnGroupTest -->
      <class name="annotations.AnnotationTest">
        <test-method is-config="true" signature="beforeSuite()[pri:0, instance:annotations.AnnotationTest@5dd6264]" started-at="2020-11-26T22:37:05 MMT" name="beforeSuite" finished-at="2020-11-26T22:37:05 MMT" duration-ms="69" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method is-config="true" signature="beforeTest()[pri:0, instance:annotations.AnnotationTest@5dd6264]" started-at="2020-11-26T22:37:05 MMT" name="beforeTest" finished-at="2020-11-26T22:37:05 MMT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:annotations.AnnotationTest@5dd6264]" started-at="2020-11-26T22:40:10 MMT" name="beforeClass" finished-at="2020-11-26T22:40:10 MMT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:annotations.AnnotationTest@5dd6264]" started-at="2020-11-26T22:40:10 MMT" name="beforeMethod" finished-at="2020-11-26T22:40:10 MMT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="login()[pri:0, instance:annotations.AnnotationTest@5dd6264]" started-at="2020-11-26T22:40:10 MMT" name="login" finished-at="2020-11-26T22:40:10 MMT" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:annotations.AnnotationTest@5dd6264]" started-at="2020-11-26T22:40:10 MMT" name="afterMethod" finished-at="2020-11-26T22:40:10 MMT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:annotations.AnnotationTest@5dd6264]" started-at="2020-11-26T22:40:10 MMT" name="beforeMethod" finished-at="2020-11-26T22:40:10 MMT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="searchProduct()[pri:0, instance:annotations.AnnotationTest@5dd6264]" started-at="2020-11-26T22:40:10 MMT" name="searchProduct" finished-at="2020-11-26T22:40:10 MMT" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- searchProduct -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:annotations.AnnotationTest@5dd6264]" started-at="2020-11-26T22:40:10 MMT" name="afterMethod" finished-at="2020-11-26T22:40:10 MMT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:annotations.AnnotationTest@5dd6264]" started-at="2020-11-26T22:40:10 MMT" name="afterClass" finished-at="2020-11-26T22:40:10 MMT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method is-config="true" signature="afterTest()[pri:0, instance:annotations.AnnotationTest@5dd6264]" started-at="2020-11-26T22:43:51 MMT" name="afterTest" finished-at="2020-11-26T22:43:51 MMT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method is-config="true" signature="afterSuite()[pri:0, instance:annotations.AnnotationTest@5dd6264]" started-at="2020-11-26T22:43:54 MMT" name="afterSuite" finished-at="2020-11-26T22:43:54 MMT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- annotations.AnnotationTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
